templates/demo-store/package.json

```diff
@@ -4,7 +4,7 @@
   "sideEffects": false,
   "version": "2.1.6",
   "scripts": {
-    "dev": "shopify hydrogen dev --worker --codegen",
+    "dev": "cross-env LOCAL_DEV=true shopify hydrogen dev --worker --codegen",
```

Comment:
We still support `h2 init --template demo-store` and we don't want LOCAL_DEV in user apps. Can you maybe create a script for this in the root package like the existing `dev:app` for skeleton?

Reply:
removed